src/main/webapp/util/Utils.js

```diff
@@ -45,3 +48,13 @@ export const scrollToTop = () => {
 export const handleRedirectToProject = (serverURL, projectKey) => {
   return [serverURL, "project", "main?projectKey=" + projectKey + "&tab=review"].join("/");
 }
+
+export const obtainStringRole = (role) => {
+  return USER_ROLES.map(it => it.value === role);
+}
+
+export const isCurrentUserAdmin = () =>  {
```

Comment:
As I see usages of this, why don't we change the API call that gets the user to calculate this on the server side and add it to the user object? We call this method so many times, it seems very unnecessary. Every page should have the user in context, so if this value were already on the user object, we would not have to make an additional query to get it.

Reply:
Answering my own question here. We already do this. See `AuthenticatedController.groovy`:
```
    def getSessionUser() {
        User user = getUser()
        render([
            'displayName': user.displayName,
            'emailAddress': user.emailAddress,
            'userName': user.userName,
            'isAdmin': isAdmin(),
            'isORSP': isORSP(),
            "isViewer": isViewer(),
            'isComplianceOffice': isComplianceOffice(),
        ] as JSON)
    }
```
The user object we get back from the server already knows if the user is an admin already. I think this method, and all callers of it, should be removed and look at the user directly.

Reply:
The main idea of ​​creating a helper in the ui was to get these session permission to unify and similar roles `orsp` and` admin`, but I see that we can save of doing this by using `isAdmin ()` from groovy's `SupplementalRole` class . 
I had to do a little [change](https://github.com/broadinstitute/orsp-pub/pull/178/files#diff-56bd5adea21ec7add82600bafad8443eR12) there due to, as mentioned above, Orsp and Admin are the same, at least for now, so if you call `isAdmin()` it verifies if the user has admin or orsp roles.